### DISCUSSION TITLE
Unable to get incoming mails - Broken pipe smtp-server

### DISCUSSION THREAD

**Discussion by RahulVerma989** [postalserver/postal#2861] [Help with using Postal]
Hi there,

I am able to receive emails using the **Simulate an incoming e-mail** option but not able to receive when sending through gmail.

After executing postal logs smtp I am getting these logs.

<img width="1288" alt="Screenshot 2024-03-09 at 3 17 50 PM" src="https://github.com/postalserver/postal/assets/48805821/687b5e1b-b4dd-4be4-92e4-5c6aa576aa5e">

My domain is also setup correctly.

<img width="1193" alt="Screenshot 2024-03-09 at 3 20 48 PM" src="https://github.com/postalserver/postal/assets/48805821/fbd29146-3bfc-4570-a2a8-8154778ab01e">

How should I solve this issue?

**Comment by adamcooke** [comment]
There's something amiss with your configuration. You say that you're sending the email from Gmail but the SMTP connection is coming from `::1` (localhost) rather than an external mail server.
